0001-interledger-architecture/0001-interledger-architecture.md

```diff
@@ -1,252 +1,135 @@
+A connector receives a local transfer on one ledger in exchange for making another local transfer on a different ledger. A single interledger payment may include multiple connectors and may traverse any number of ledgers.
 
-Note that this does not mean that all connectors are necessarily stateless. They may keep track of their available liquidity or even maintain an internal ledger if they are transacting on behalf of third parties. But we are assuming that they do not keep authoritative state about transfers.
+If the ledgers represent different assets, the connectors set the exchange rate between the transfers. Connectors may generate revenue from the difference in value between incoming and outgoing transfers. Senders may request quotes from multiple connectors to determine the best price before sending a payment.
 
-#### Routing complexity should be in the connectors.
+Connectors *peer* with one another to exchange information used to determine the best route for a payment.
 
-Routing is a complex and difficult problem, and ought to be performed by the connectors, not the end users of the system. An important objective is to isolate user software from changes caused by the inevitable evolution of the Interledger routing architecture.
+**Interledger ensures that senders' funds are safe throughout an multi-hop payment and cannot be stolen by faulty or malicious connectors (see [Interledger Security](#interledger-security)).**
 
-#### The system must tolerate wide ledger variation.
+### The Interledger
 
-A basic objective of the Interledger design is to tolerate a wide range of ledger characteristics --- e.g. throughput, latency, fees, access restrictions, reliability, decentralization. In addition it must support all types of different asset characteristics --- e.g. divisibility, fungibility, interest/demurrage, issuance.
+The Interledger protocol suite may be used to transact across any ledgers and connectors, whether they are public or private. There is no single network that all parties must connect to to use these protocols.
 
-### Interledger Protocol Suite
+"The Interledger" is a conceptual network made up of independent and diverse ledgers linked by connectors. Each account "on the Interledger" is part of a particular ledger, but they may transact with others by sending interledger payments through different ledgers and connectors. Like "the Internet", the Interledger is not a single network but is comprised of multiple interconnected networks.
 
-The Interledger architecture is separated into four layers:
+## Interledger Security
 
-![Interledger architecture layers](../shared/graphs/interledger-model.svg)
+**Interledger uses *conditional transfers* to secure payments across multiple hops and even through untrusted connectors.**
+Senders are guaranteed proof that the receiver got the payment or their money back.
```

Comment:
Without a blank line between, this is interpreted by most Markdown parsers as a hard-wrapped continuation of the previous sentence's paragraph. You should probably set it off with a separate line, and maybe even make it into bullet points.

Also, we can strengthen this sentence:

> Senders are guaranteed proof that the receiver got the payment or their money back.

Suggested rephrase:

> Senders are guaranteed cryptographic proof that the receiver got the payment or their money back, no matter how many ledgers and connectors are in between.